src/renderer/dx/DxRenderer.cpp

```diff
@@ -860,15 +869,19 @@ void DxEngine::_InvalidOr(RECT rc) noexcept
                                 _glyphCell.cx);
 
         // Get the baseline for this font as that's where we draw from
-        DWRITE_LINE_SPACING spacing;
-        RETURN_IF_FAILED(_dwriteTextFormat->GetLineSpacing(&spacing));
+        DWRITE_LINE_SPACING_METHOD spacingMethod;

```

Comment:
Why can't you just fill the members of the DWRITE_LINE_SPACING struct with the relevant bits instead of making this 3 parameters now?